frontend/src/libs/components/meditation-sidebar/styles.module.scss

```diff
@@ -0,0 +1,66 @@
+.container {
+  width: 350px;
+  height: 100%;
+  background: linear-gradient(
+    to bottom,
+    var(--transparent) 0,
+    var(--blue-100) 100%
+  );
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 10%;
+  padding: 0 20px;
+  font-size: 20px;
+  border-bottom: 1px solid var(--blue-100);
+}
+
+.info {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 90%;
+}
+
+.list .search {
+  margin-top: 23px;
+}
+
+.list .meditation-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  margin-top: 83px;
+  overflow-y: scroll;
+}
+
+.meditation-list::-webkit-scrollbar {
+  display: none;
+}
+
+@media (width <= 480px) {
```

Comment:
```suggestion
@media (width < 480px) {
```

Reply:
@Nanaier but you wrote the same in this commit: https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/commit/6889f8576e799b4d06f272a415023973945c501a#diff-3a9fe4b436ce13ce1175189015f6dab5498b3a528e277cf353310c48e131dbd4
So is it incorrect?

Reply:
done